packages/cli/src/commands/up.ts

```diff
@@ -51,73 +52,82 @@ export default class Up extends MachineCreationDriverCommand<typeof Up> {
       insecureSkipVerify: flags['insecure-skip-verify'],
     }
 
-    const { hostKey, clientId, rootUrl } = await tunnelServerHello({
-      tunnelingKey,
-      knownServerPublicKeys: pStore.knownServerPublicKeys,
-      tunnelOpts,
-      log: this.logger,
+    await withSpinner(async spinner => {
```

Comment:
spinner and log don't work together well, because the logs print a newline which is left behind on the next "frame" of the spinner.

also - the spinner should only wrap the `Initializing tunnel connection...` action, not the whole flow.